src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.OSX/AppleCertificatePal.cs

```diff
@@ -109,7 +111,20 @@ internal sealed partial class AppleCertificatePal : ICertificatePal
 
                 using (keychain)
                 {
-                    return ImportPkcs12(rawData, password, exportable, keychain);
+                    var  ret = ImportPkcs12(rawData, password, exportable, keychain);
```

Comment:
```suggestion
                    ICertificatePal ret = ImportPkcs12(rawData, password, exportable, keychain);
```

Or alternatively `AppleCertificatePal` if you decide to change the return type.

Reply:
I don't know if the cast matters but I update it to return AppleCertificatePal.